src/neo/SmartContract/NefFile.cs

```diff
@@ -57,6 +57,8 @@ public class NefFile : ISerializable
             (sizeof(int) * 4) + // Version
             UInt160.Length;     // ScriptHash
 
+        public static readonly int MaxLength = HeaderSize + sizeof(uint) + IO.Helper.GetVarSize(1024 * 1024) + (1024 * 1024);
```

Comment:
```suggestion
        public const int MaxLength = 1024 * 1024;
```

Reply:
Removed because it's duplicated on `ApplicationEngine`

Reply:
But sometimes we read NEF files directly.

Reply:
Done